clientv3/watch.go

```diff
@@ -188,7 +188,7 @@ type watcherStream struct {
 }
 
 func NewWatcher(c *Client) Watcher {
-	return NewWatchFromWatchClient(pb.NewWatchClient(c.conn))
+	return NewWatchFromWatchClient(RetryWatchClient(c))
```

Comment:
let us keep watch to use its own retry as it is. do not remove failfast. keep it as a todo?